core/cmd/cosmos_node_commands_test.go

```diff
@@ -52,6 +52,7 @@ func TestShell_IndexCosmosNodes(t *testing.T) {
 	assert.Equal(t, chainID, n.ChainID)
 	assert.Equal(t, *node.Name, n.ID)
 	assert.Equal(t, *node.Name, n.Name)
+	assert.Contains(t, n.Config, node.TendermintURL.Host)
```

Comment:
```suggestion
	assert.Contains(t, n.Config, node.TendermintURL.String())
```

Reply:
Oh you can cast it in that case:
```suggestion
	assert.Contains(t, n.Config, (*url.URL)(node.TendermintURL).String())
```

Reply:
Or maybe a cleaner approach for these is to marshal the input config as TOML, and just compare that directly against `n.Config`?

Reply:
Just add the `String()` method to `utils.URL` no?

Reply:
Added some more tests to check if the order of the fields is correct since that was the cause of the bug, `n.Config` is already checked for equality so no need for extra checks. [`3bf165f` (#9558)](https://github.com/smartcontractkit/chainlink/pull/9558/commits/3bf165f799a29a401c4a1d4ef08cabb4cdb15a1d)